frontend/src/entry-forms/PostingMetadata.svelte

```diff
@@ -0,0 +1,107 @@
+<script lang="ts">
```

Comment:
This seems to be nearly an identical copy of the corresponding component for entries - please reuse that one